src/renderer/redux/actions/content.js

```diff
@@ -347,6 +364,42 @@ export function doPurchaseUri(uri) {
   };
 }
 
+export function doFetchFeaturedChannels(uris) {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.FETCH_FEATURED_CHANNELS_STARTED
+    });
+
+    Lbry.claim_list_by_channel({uris, page: 1})
```

Comment:
I think if you use `doFetchClaimsByChannel` you can eliminate most or a lot of this code.